src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -1272,3 +1276,49 @@ def remove(path: str):
             ws.dbfs.delete(path, recursive=True)
 
     yield from factory("make_storage_dir", create, remove)
+
+
+@pytest.fixture
+def make_dashboard(ws, make_random, make_query):
+    def create() -> Dashboard:
+        query = make_query()
+        viz = ws.query_visualizations.create(
+            type="table",
+            query_id=query.id,
+            options={
+                "itemsPerPage": 1,
+                "condensed": True,
+                "withRowNumber": False,
+                "version": 2,
+                "columns": [
+                    {"name": "id", "title": "id", "allowSearch": True},
+                ],
+            },
+        )
+
+        dashboard_name = f"ucx_dashboard_D{make_random(4)}"
```

Comment:
More consistent with the other resources created in ucx fixtures

```suggestion
        dashboard_name = f"ucx_D{make_random(4)}"
```